server/moodle/mod/livequiz/classes/services/livequiz_services.php

```diff
@@ -166,4 +205,19 @@ private function get_questions_with_answers(int $quizid): array {
         }
         return $questions;
     }
+    /**
+     * Creates a new participation record in the database.
+     * @param int $studentid
+     * @param int $quizid
+     * @throws dml_exception
```

Comment:
This function doesnt throw a dml_exception?

Reply:
This is our codesniffer requesting us to have it, do not know why

Reply:
You'll see that in the newest commits there's a transaction being started, that might throw a dml_exception.